src/flattenHelper.ts

```diff
@@ -43,6 +43,9 @@ export class FlattenHelper {
         delete (<any>vp).flattenedNames;
 
         vp.language = JSON.parse(JSON.stringify(vp.language));
+        // copied object does not belong to original code model, so has no path
+        delete vp.language?.['cli']?.[NodeCliHelper.CLI_NODE_PATH];
```

Comment:
move this to copyHelper?

Reply:
Basically I think we are creating a `VirtualParameter` based on `Parameter`, I think it's not a copy. I prefer to not move it to copyHelper ;-)

Reply:
I mean cloneLanguage()

---
In reply to: [448995788](https://github.com/Azure/autorest.clicommon/pull/79#discussion_r448995788) [](ancestors = 448995788)

Reply:
Ok, added copyLanguage()